README.md

```diff
@@ -4,42 +4,42 @@ wpantund, Userspace WPAN Network Daemon
 `wpantund` is a user-space network interface driver/daemon that
 provides a native IPv6 network interface to a low-power wireless
 **Network Co-Processor** (or *NCP*). It was written and developed by
-Nest Labs to make supporting [Thread](http://threadgroup.org)
-connectivity on unix-like operating systems more straightforward.
+Nest to support [Thread](http://threadgroup.org)
+connectivity on uUnix-like operating systems and make the support more straightforward.
 
-`wpantund` is designed to marshall all access to the NCP, ensuring
-that it always remains in a consistent, well-defined state.
+`wpantund` is designed to marshall all access to the NCP ensuring
+that it always remains in a consistent and well-defined state.
 
 This is not an official Google product.
 
-## Feature and Architecture Summary ##
+## Feature and architecture summary ##
```

Comment:
Keep the original. We have a new lead tech writer, she will serve as manager, she came from VMWare, so I think she will let us do what we want . . . within reason.